test/validator/index.js

```diff
@@ -2,7 +2,7 @@ import * as fs from "fs";
 import assert from "assert";
 import { svelte, tryToLoadJson } from "../helpers.js";
 
-describe("validate", () => {
+describe.only("validate", () => {
```

Comment:
You have an only here

Reply:
yeah, forgot to remove it before pushing — it's just to make my life easier while I'm working on it (as I don't need to configure tests to run individually, or wait for the entire suite). will make sure I remove it before removing the WIP 😀